source/deploy/mobile-app-changelog.md

```diff
@@ -2,12 +2,51 @@
 
 Latest Mattermost Mobile Apps releases:
 
-- [2.1.0 Release](#release)
-- [2.0.1 Release](#id1)
-- [2.0.0 Release](#id6)
-- [1.55.1 Release](#id9)
-- [1.55.0 Release](#id13)
-- [1.54.0 Release](#id18)
+- [2.2.0 Release](#release)
+- [2.1.0 Release](#id1)
+- [2.0.1 Release](#id6)
+- [2.0.0 Release](#id11)
+- [1.55.1 Release](#id14)
+- [1.55.0 Release](#id18)
+
+## 2.2.0 Release
+- Release Date: March 16, 2023
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added the ability to set the mobile notifications preferences per channel.
+ - Calls: Added slash commands to start/stop call recordings (``/call recording [start|stop]``).
+ - Calls: Implemented a glare free negotiation.
```

Comment:
Curious to know more about what this means

Reply:
It's a problem with Webrtc where if two people try to simultaneously connect to each other, the negotiation fails (they can't fully connect).

Reply:
Thanks, @cpoile! Do we expect end users to be familiar with the term "glare" in this context? If not, we may want to use language more familiar to the reader.

Reply:
Maybe: 
```suggestion
 - Calls: Implemented glare free negotiation. This fix prevents potential negotiation problems when two clients try to connect simultaneously.
```